### PR TITLE
Update ColumnPicker to use GWT.create allowing proper proxy creation …

### PR DESCRIPTION
**Pull request by cristianonicolai**
…when using GWTMockito

### CODE REVIEW

File: uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/ColumnPicker.java [hunk 1]
```diff
@@ -210,7 +214,8 @@ public void onValueChange( ValueChangeEvent<Boolean> booleanValueChangeEvent ) {
         }
 
         if ( gridPreferences != null ) {
-            Button resetButton = new Button( "Reset" );
```
Comment (ederign) [inline uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/ColumnPicker.java]: i18n?

**Comment by ederign** [comment]
Just a concern about i18n. If this isn't necessary +1.

**Comment by cristianonicolai** [comment]
@ederign thanks! Easy to fix ;)

**Comment by csadilek** [comment]
0.8.x: f8d2128c8e1d7191770fd7eabfdb12b8debe393a

**Pull request closed without merging**